src/components/Single-Article.jsx

```diff
@@ -34,9 +39,21 @@ export const SingleArticle = () => {
                 <p>Comments: {article[0].comment_count}</p>
                 <p>Date Posted: {article[0].created_at}</p>
                 </div> 
-
             )}
         </div>
+
+        <div className ="comment">
+            {comments.map((comment) => {
+            return (
+            <div>
+            <p>Author: {comment.author}</p>
```

Comment:
Good work so far - see this article for more interesting ways of displaying information: https://www.refactoringui.com/previews/labels-are-a-last-resort